src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -123,89 +221,62 @@ public static class DnnCatalog
-            string modelSavePath = null,
-            string finalModelPrefix = "custom_retrained_model_based_on_",
-            IDataView validationSet = null,
-            bool testOnTrainSet = true,
-            bool reuseTrainSetBottleneckCachedValues = false,
-            bool reuseValidationSetBottleneckCachedValues = false,
-            string trainSetBottleneckCachedValuesFilePath = "trainSetBottleneckFile.csv",
-            string validationSetBottleneckCachedValuesFilePath = "validationSetBottleneckFile.csv"
+            IDataView validationSet = null
             )
         {
             var options = new ImageClassificationEstimator.Options()
             {
-                ModelLocation = ModelLocation[arch],
                 InputColumns = new[] { featuresColumnName },
                 OutputColumns = new[] { scoreColumnName, predictedLabelColumnName },
                 LabelColumn = labelColumnName,
                 TensorFlowLabel = labelColumnName,
-                Epoch = epoch,
-                LearningRate = learningRate,
-                BatchSize = batchSize,
-                EarlyStoppingCriteria = disableEarlyStopping ? null : earlyStopping == null ? new EarlyStopping() : earlyStopping,
+                Epoch = 100,
+                EarlyStoppingCriteria = new EarlyStopping(),
                 ScoreColumnName = scoreColumnName,
                 PredictedLabelColumnName = predictedLabelColumnName,
-                FinalModelPrefix = finalModelPrefix,
-                Arch = arch,
-                MetricsCallback = metricsCallback,
-                StatisticsFrequency = statisticFrequency,
-                Framework = framework,
-                ModelSavePath = modelSavePath,
+                FinalModelPrefix = "custom_retrained_model_based_on_",
+                Arch = Architecture.InceptionV3,
+                MetricsCallback = null,
                 ValidationSet = validationSet,
-                TestOnTrainSet = testOnTrainSet,
-                TrainSetBottleneckCachedValuesFilePath = trainSetBottleneckCachedValuesFilePath,
-                ValidationSetBottleneckCachedValuesFilePath = validationSetBottleneckCachedValuesFilePath,
-                ReuseTrainSetBottleneckCachedValues = reuseTrainSetBottleneckCachedValues,
-                ReuseValidationSetBottleneckCachedValues = reuseValidationSetBottleneckCachedValues
+                TestOnTrainSet = true,
+                TrainSetBottleneckCachedValuesFilePath = "trainSetBottleneckFile.csv",
+                ValidationSetBottleneckCachedValuesFilePath = "validationSetBottleneckFile.csv",
+                ReuseTrainSetBottleneckCachedValues = false,
+                ReuseValidationSetBottleneckCachedValues = false
             };
 
-            if (!File.Exists(options.ModelLocation))
+            var env = CatalogUtils.GetEnvironment(catalog);
+            return new ImageClassificationEstimator(env, options, DnnUtils.LoadDnnModel(env, options.Arch, true));
```

Comment:
This method should just call into the `ImageClassification` overload below that takes an `Options` class.